jx_sqlite/query_table.py

```diff
@@ -345,8 +345,90 @@ def query(self, query):
         return output
 
     def query_metadata(self, query):
-        Log.error("Not implemented yet")
+        frum, query['from'] = query['from'], self
+        schema = self.sf.tables["."].schema
+        query = QueryOp.wrap(query, schema)
+
+        if query.edges or query.groupby:
+            Log.error("Aggregates(groupby or edge) are not supported")
+        where = query.where
+        tableName = None
+        columnName = None
+        if where.op == "eq" and where.lhs.var == "table":
+            tableName = mo_json.json2value(where.rhs.json)
+        elif where.op =="eq" and where.lhs.var =="name":
+            columnName = mo_json.json2value(where.rhs.json)
+        else:
+            Log.error("Only simple filters are expected like: \"eq\" on table and column name")
+        
+        metadata = []
+        result = self.db.query("""SELECT NAME FROM SQLITE_MASTER WHERE type='table' ORDER BY NAME;""")
```

Comment:
... given every column in a schema belongs to every table, then asking the database for this may not be easy.  The `snowflake.Schema` should already have all the information we need; just it must be filtered down to include just the table and columns required.  Maybe pure python is best.